src/centos/scripts/start-marklogic.sh

```diff
@@ -147,6 +147,49 @@ function restart_check {
     error "Failed to restart $1" exit
 }
 
+################################################################
+# setting default protocol and CA cert for curl calls to bootstrap host
+################################################################
+ML_BOOTSTRAP_PROTOCOL="http"
+ML_CACERT_FILE="/run/secrets/${MARKLOGIC_JOIN_CACERT_FILE}"
+
+################################################################
+# Check if TLS is enabled on Bootstrap host
```

Comment:
Missing the parameter for the function document

Reply:
added params for func documentation